### DISCUSSION TITLE
⭐️ Starstruck Exchange ⭐️

### DISCUSSION THREAD

**Discussion by xommmax** [Schweinepriester/achievement-testarea#26] [Q&A]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰
After you starred leave a comment here with your repo and I will star it in response

**Comment by nguyen-tuan-hiep** [comment]
I just star your repo, can you star mine: https://github.com/nguyen-tuan-hiep/Introduction-to-Communication-Engineering

**Reply by 1wos** [reply]
I marked your repo with stars! Can you do it for me, too?
https://github.com/sommizzu/Big-Data-Convergence

**Reply by ghost** [reply]
done, https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Reply by Rantoryu** [reply]
@sommizzu and @nguyen-tuan-hiep  I've starred yours.
Please, star my repo too :)
https://github.com/Rantoryu/Personal_Website

**Reply by michalm57** [reply]
@sommizzu @nguyen-tuan-hiep i gave you stars, please give me stars too. :)
https://github.com/michalm57/currency_exchange

**Reply by pulkitxm** [reply]
did it for you , give mine as well please
https://github.com/Pulkitxm/Delta-Apna-College

**Reply by Mattral** [reply]
Done!

Please mine too 
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by 1wos** [reply]
@Mattral  I marked your repo with stars! Can you do it for me, too?
https://github.com/sommizzu/Big-Data-Convergence

**Reply by Mattral** [reply]
Done!
That was nice dataset!

**Reply by Mattral** [reply]
I followed you.
lets connect

**Reply by ganochenkodg** [reply]
@nguyen-tuan-hiep done, star my repo pls https://github.com/ganochenkodg/qdrant-operator

**Reply by AdityaNow** [reply]
@nguyen-tuan-hiep
Done, please star my repo https://github.com/AdityaNow/NavtorAssignment
Help me get the badge ⭐
Thank you.

**Comment by 1wos** [comment]
I marked your repo with stars! Can you do it for me, too?
https://github.com/sommizzu/Big-Data-Convergence

**Reply by ghost** [reply]
done, https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Comment by arpittyagi102** [comment]
I marked your repo with stars! Can you do it for me, too?
https://github.com/arpittyagirocks/Hideandseek

**Reply by ghost** [reply]
@arpittyagirocks @sommizzu @nguyen-tuan-hiep @4lfant 
starred all, please star back my repo :
https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Reply by Rantoryu** [reply]
@chriskthomas and @arpittyagirocks 
I've starred your repos. Please, star my repo too :)
https://github.com/Rantoryu/Personal_Website

**Reply by arpittyagi102** [reply]
Done! 
@Rantoryu

**Reply by michalm57** [reply]
@arpittyagirocks, @chriskthomas i gave you stars, please give me stars too. :)
https://github.com/michalm57/currency_exchange

**Reply by pulkitxm** [reply]
did it for you , give mine as well please
https://github.com/Pulkitxm/Delta-Apna-College

**Reply by ganochenkodg** [reply]
@arpittyagi102 done, and my pls https://github.com/ganochenkodg/qdrant-operator

**Comment by michalm57** [comment]
I marked your repo with stars! Can you do it for me, too?
https://github.com/michalm57/currency_exchange

**Reply by 1wos** [reply]
I marked your repo with stars! Can you do it for me, too?
https://github.com/sommizzu/Big-Data-Convergence

**Reply by pulkitxm** [reply]
did it for you , give mine as well please
https://github.com/Pulkitxm/Delta-Apna-College

**Reply by Mattral** [reply]
Done!

Please mine too 
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Comment by pulkitxm** [comment]
star my  [repo](https://github.com/Pulkitxm/Delta-Apna-College), I'll do it for yours
 [https://github.com/Pulkitxm/Delta-Apna-College](https://github.com/Pulkitxm/Delta-Apna-College)
Have a nice Day !

**Reply by 1wos** [reply]
I'm done with your repo star! Can you do mine, too?
https://github.com/sommizzu/Big-Data-Convergence

**Reply by pulkitxm** [reply]
I'm also done ! Thanks

**Reply by Rantoryu** [reply]
@sommizzu I've starred yours. Can You star mine too?
https://github.com/Rantoryu/Personal_Website

**Comment by Rantoryu** [comment]
@4lfant I've starred your repo. Can You do one for me?
https://github.com/Rantoryu/Personal_Website

**Comment by ternera** [comment]
Please star and I will return it: https://github.com/ternera/ternera.org

**Reply by michalm57** [reply]
@ternera done :), here is my repo:
https://github.com/michalm57/currency_exchange

**Comment by Mattral** [comment]
Done!

Please mine too 
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Comment by AdityaNow** [comment]
Done 😃 (feel free to star any of my repo https://github.com/AdityaNow/NavtorAssignment & https://github.com/AdityaNow/Wikipedia)
Help me get the badge :star:
Thank you

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Reply by 1wos** [reply]
I stared your repo can you do mine, too ? 
https://github.com/sommizzu/ASF-DataAnalysis

**Comment by winstontsui** [comment]
Can you help me star? I will return it

**Reply by AdityaNow** [reply]
Mention your repo! so that we can star it..

Star mine too - https://github.com/AdityaNow/NavtorAssignment/

**Reply by winstontsui** [reply]
Yes! It is https://github.com/winstontsui/leetcode-problems

**Reply by winstontsui** [reply]
I just starred your repo!

**Reply by AdityaNow** [reply]
starred your repo too! Thank you

**Comment by pulkitxm** [comment]
Please star my repository and I will return it: https://github.com/Pulkitxm/fullstackopen

**Reply by ganochenkodg** [reply]
Done, and my pls https://github.com/ganochenkodg/qdrant-operator

**Reply by AdityaNow** [reply]
Done, star mine too - https://github.com/AdityaNow/NavtorAssignment/

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.

**Reply by AdityaNow** [reply]
Done, please star mine too - https://github.com/AdityaNow/NavtorAssignment/

Thank you
